### PR TITLE
Add role for nsupdate_web

### PR DESCRIPTION
**Pull request by zmc**
See https://github.com/zmc/nsupdate-web/

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
Next step is probably an nginx proxy to front the server so it can listen on the main interface - but I need to make changes to `nsupdate-web` to get it to enforce a policy first.

Oh and a README for the role.

**Comment by djgalloway** [comment]
You could probably get it to pull the key from `ansible/secrets/nameserver.yml` if you wanted to.

**Comment by zmc** [comment]
@djgalloway I thought about that but I'm leaning towards that being a not-so-great idea since it'd allow anyone with the key to modify records on any of the dynamic zones. The purpose of the web service is to keep everything locked down

File: roles/nsupdate_web/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  user:
+    name: "{{ nsupdate_web_user }}"
+    state: present
+    system: true
+    shell: "/bin/false"
+
+- name: Clone nsupdate_web repo
+  git:
+    repo: "{{ repo_url }}"
+    dest: "~/nsupdate_web"
```
Comment (djgalloway) [inline roles/nsupdate_web/tasks/main.yml]: Will the git module create this dir if it doesn't exist?  Ansible docs don't say.

Comment (zmc) [inline roles/nsupdate_web/tasks/main.yml]: Yes, it will

**Comment by zmc** [comment]
@djgalloway  This now fronts with nginx. Do you think a readme is necessary?

**Comment by djgalloway** [comment]
I'd say yes.  Mainly to mention that this is a first-time setup role and not really an ongoing config management role.

Also what to do after the role has been run (like putting the keys in place) would be helpful.

**Comment by zmc** [comment]
@djgalloway repushed with a readme and the missing nginx file!

**Pull request merged**